ocamltest/builtin_actions.ml

```diff
@@ -285,6 +285,14 @@ let no_tsan = make
      "tsan not available"
      "tsan available")
 
+let no_address_sanitizer = make
```

Comment:
Is this test predicate necessary? You can also disable asan with a compiler flag `-fno-asan`